src/utils/uemParser.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { l10n } from 'vscode';
+import { Field } from './orgUtils';
+
+export class UEMParser {
+    public static findFieldValues(json: Object, targetField: String): string[] {
+        const results: string[] = [];
+
+        function search(json: any) {
```

Comment:
Why do you have a function within a function here? I would either inline all of this functionality, or pull out a helper method from the wrapping method, so that it can be more easily tested in isolation.